docs/game_sdk/Networking.md

```diff
@@ -206,51 +229,89 @@ A quick note here may be helpful for the two functions that should be called con
 
 `Flush()` is specific to the network manager. It actually _writes_ the packets out to the stream. You should call this at the _end_ of your game loop as a way of saying "OK, I'm done with networking stuff, go send all the stuff to people who need it". In Unity, for example, this goes in `LateUpdate()`.
 
+## Connecting to Each Other
+
+This manager is built around the concept of routes between players, and then channels on those routes. Player A opens a route to Player B. This route will often change as our scaled networking system works to find the best networking path between those players. As that route changes, the player will receive `OnRouteUpdate` events. They should then alert other lobby members that their route has changed by updating their lobbymetadata. Other lobby members will see those updates from the `OnLobbyMemberUpdate` event, and can call `UpdateRoute()` accordingly. A user's route could change frequently, so architect your system anticipating frequent route changes and updates.
```

Comment:
Typically a user's route will only change if their external IP address changes, either because their ISP changed it or because they switched networks (e.g. between Ethernet and Wifi).

Reply:
Leave the warning about a user's route changing frequently though, because it *can* change frequently in some weird edge cases, like users connected to networks that load balance between multiple external ISPs.